File: app/src/main/java/com/rkbapps/neetflix/models/tvseries/seasons/SeasonsDetails.java
package com.rkbapps.neetflix.models.tvseries.seasons;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;


public class SeasonsDetails {


    @SerializedName("air_date")
    @Expose
    private String airDate;
    @SerializedName("episodes")
    @Expose
    private List<EpisodeDetails> episodes;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("overview")
    @Expose
    private String overview;
    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("poster_path")
    @Expose
    private String posterPath;
    @SerializedName("season_number")
    @Expose
    private Integer seasonNumber;

    public String getAirDate() {
        return airDate;
    }

    public List<EpisodeDetails> getEpisodes() {
        return episodes;
    }

    public String getName() {
        return name;
    }

    public String getOverview() {
        return overview;
    }

    public Integer getId() {
        return id;
    }

    public String getPosterPath() {
        return posterPath;
    }

    public Integer getSeasonNumber() {
        return seasonNumber;
    }
}

File: app/src/main/java/com/rkbapps/neetflix/models/tvseries/seasons/SeasonsDetails.kt
package com.rkbapps.neetflix.models.tvseries.seasons

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class SeasonsDetails(
    @SerializedName("air_date")
    @Expose
    val airDate: String,

    @SerializedName("episodes")
    @Expose
    val episodes: List<EpisodeDetails>,

    @SerializedName("name")
    @Expose
    val name: String,

    @SerializedName("overview")
    @Expose
    val overview: String,

    @SerializedName("id")
    @Expose
    val id: Int,

    @SerializedName("poster_path")
    @Expose
    val posterPath: String?,

    @SerializedName("season_number")
    @Expose
    val seasonNumber: Int,
)